rasa/cli/interactive.py

```diff
@@ -32,7 +32,7 @@ def add_subparser(
         conflict_handler="resolve",
         parents=parents,
         formatter_class=argparse.ArgumentDefaultsHelpFormatter,
-        help="Train a Rasa Core model with interactive learning",
+        help="Trains a Rasa Core model in an interactive learning session.",
```

Comment:
Does this train the model or just create new training data as above?
Also, maybe something that says ```Uses the RegexInterpreter, i.e. `/intent` input format``` or something like that